### ISSUE TITLE
install.packages("RLumShiny")   error

### ISSUE THREAD

**Issue by 1457634627** [venyao/shinyCircos#12]
XMLTree.c: In function ‘xmlBufferWrite’:
XMLTree.c:1394:9: error: void value not ignored as it ought to be
     ret = xmlBufferAdd((xmlBufferPtr) context, (const xmlChar *) buffer, len);
         ^
make: *** [/usr/lib64/R/etc/Makeconf:172: XMLTree.o] Error 1
ERROR: compilation failed for package ‘XML’
* removing ‘/usr/lib64/R/library/XML’
ERROR: dependency ‘XML’ is not available for package ‘Luminescence’
* removing ‘/usr/lib64/R/library/Luminescence’
ERROR: dependency ‘Luminescence’ is not available for package ‘RLumShiny’
* removing ‘/usr/lib64/R/library/RLumShiny’

The downloaded source packages are in
	‘/tmp/RtmpLadHqx/downloaded_packages’
Updating HTML index of packages in '.Library'
Making 'packages.html' ... done
Warning messages:
1: In install.packages("RLumShiny") :
  installation of package ‘XML’ had non-zero exit status
2: In install.packages("RLumShiny") :
  installation of package ‘Luminescence’ had non-zero exit status
3: In install.packages("RLumShiny") :
  installation of package ‘RLumShiny’ had non-zero exit status

How to solve this ？

**Comment by venyao** [comment]
It seems that the problem was caused by the dependency package 'XML'. Check this page https://cran.r-project.org/web/packages/XML/index.html.

What's your system? Windows or Linux?

I guess this problem can be solved by installing libxml2 (>= 2.6.3) on your system.

**Comment by naborlozada** [comment]
Hi,
I got the same error message. The problem was the R version that I was using (3.6.2), since the package ‘Luminescence’ can be (apparently) installed only in R version 4.0 or higher.

```R
# R version 4.0.5 (2021-03-31) -- "Shake and Throw"
# $ ./R

install.packages("RLumShiny")
# ...
# ...
# * DONE (RLumShiny)
#
#The downloaded source packages are in
#	‘/tmp/RtmpAQlMyy/downloaded_packages’
>
> library("RLumShiny")
Creating a generic function for ‘toJSON’ from package ‘jsonlite’ in package ‘googleVis’
> 
```

I hope it helps for someone facing the same problem in the future.
Best.
